lib/ui/painting/codec.h

```diff
@@ -59,7 +59,7 @@ class MultiFrameCodec : public Codec {
   int nextFrameIndex_;
 
   std::vector<SkCodec::FrameInfo> frameInfos_;
-  std::vector<SkBitmap> frameBitmaps_;
+  std::map<int, SkBitmap> frameBitmaps_;
```

Comment:
nit: would love a comment here explaining that we only keep frames that have dependent frames on them in this map.